aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,36 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+
+## January 24 - February 11, 2022
+
+### Hpe theme breaking change
+**Text-Weak** - Added a new Text color for `text-weak` per request there was a need for a new Gray text color. The new "weak" text is #757575 which passes accessibility for AA. It will be the new "weak" text, but will NOT be disabled. The previous text "weak" will now become the new text "xweak".
```

Comment:
Is this change only for 'light-mode' values? Any affects to dark mode?

```suggestion
**Text-Weak** - To meet accessibility requirements, the color value for `text-weak` is changing from `#BBBBBB` to `#757575`. A new namespace `text-xweak` is being added with the value `#BBBBBB`.

The new `text-weak` value passed WCAG AA contrast values. `#BBBBBB` will remain the value for disabled fonts.
```

Reply:
no also dark mode Ill add that